bin/configs/kotlin-spring-cloud.yaml

```diff
@@ -0,0 +1,12 @@
+generatorName: kotlin-spring
+outputDir: samples/client/petstore/kotlin-spring-cloud
+library: spring-cloud
+inputSpec: modules/openapi-generator/src/test/resources/2_0/petstore.yaml
```

Comment:
please use 3.0 spec instead: `modules/openapi-generator/src/test/resources/3_0/petstore.yaml`

Reply:
i've replaced it